src/src/PluginStructs/P116_data_struct.cpp

```diff
@@ -16,6 +16,7 @@ const __FlashStringHelper* ST77xx_type_toString(ST77xx_type_e device) {
     case ST77xx_type_e::ST7789vw_240x240: return F("ST7789 240 x 240px");
     case ST77xx_type_e::ST7789vw_240x280: return F("ST7789 240 x 280px");
     case ST77xx_type_e::ST7789vw_135x240: return F("ST7789 135 x 240px");
+    case ST77xx_type_e::ST7796s_320x480: return F("ST7796 480 x 320px");
```

Comment:
Either the enum has hor/vert resolution mixed, or the string has.

Reply:
Changed the enum, but missed the description, will fix.